environment/tplink-tapo-c100v5.uenv.txt

```diff
@@ -0,0 +1,22 @@
+day_night_color=true
+day_night_ir850=true
+day_night_ir940=false
+day_night_ircut=true
+day_night_white=false
+day_night_max=15000
+day_night_min=500
+disable_eth=true
+enable_updates=true
+gpio_button=62
+gpio_default=42o 43o 49o 50O 57o 60o 62i 63o
+gpio_ir850=49
+gpio_ircut=57o
+gpio_led_g=42
+gpio_led_r=43
```

Comment:
Please add a suffix, O or o to indicate if the leds need to be high or low to be on

Reply:
So, if for example `gpio set 42 1` lights the green led it should be `42O`?

It's copied from the other tapo env, so it needs to be updated there too.